### DISCUSSION TITLE
Having issues pushing from my terminal/ local repo to GitHub remote

### DISCUSSION THREAD

**Discussion by cy-porteous** [community/community#116674] [Repositories]
### Select Topic Area

Bug

### Body

Hello all,

I'm new to coding and using GitHub, well I've recently completed an intensive web dev bootcamp and had support from the teaching staff whenever I had issues or concerns re GH. At the moment when trying to push from my terminal I get this error despite:
- entering my correct username
- entering the correct password

➜  Excursion git:(main) git push https://github.com/cy-porteous/codeacademy_excursion
Username for 'https://github.com': cy-porteous
Password for 'https://cy-porteous@github.com':
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.

I'm used to pushing from my terminal and of course making various commit messages, at the moment I am having to add the files to the remote from my GH in the browser and this does not allow me to submit various commit messages throughout the project from my terminal. 

Any help or advice is most welcome.

Thanks,

Cy

**Comment by jmelowry** [comment]
You'll want to switch to cloning with [ssh](https://docs.github.com/en/authentication/connecting-to-github-with-ssh) 


https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent

Then, clone your repo locally again, but use the ssh uri 
![image](https://github.com/community/community/assets/2036197/e12a00a7-2cd8-4eb6-a8e5-1ffec3efc0d2)

**Reply by cy-porteous** [reply]
Thanks for the reply and pointer, trouble is that I don't use Git Bash, the steps for setting up new SSH keys suggested a Git Bash account was required?

**Reply by jmelowry** [reply]
Is Git Bash just a terminal emulator for Windows? Sorry - not very familiar. Whatever the case, no special account is needed. You generate ssh keys on the command line, and it can be done in a bash shell or powershell, etc..

**Reply by cy-porteous** [reply]
OK thanks for the tips anyway buddy, I'll have a look and if I can't find a solution I'll have to commit from the browser. Have a great day

**Comment by PurpleRain1000** [comment]
clonazione ssh
